content/docs/configuration/configuration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Configuration
+category: Configuration
+slug: 6.5.configuration
+hash: "#configuration"
+fullpath: /docs
+---
+
+Audiobookshelf is configured via environment variables.
+You can pass them to your Docker container using `-e VARIABLE=VALUE`
+or set them in `/etc/default/audiobookshelf` if you install audiobookshelf via packages.
+
+Here is a list of all available options:
+
+## Filesystem
+
+- `CONFIG_PATH` (default: `./config`)
+  - Path to the config directory.
+  - It will contain the database (users/books/libraries/settings).
+- `METADATA_PATH` (default: `./metadata`)
+  - Path to the metadata directory.
+  - It will contain cache, streams, covers, downloads, backups and logs.
+
+## External Tools
+
+- `FFMPEG_PATH` (default: `ffmpeg`)
+  - Path to the `ffmpeg` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+- `FFPROBE_PATH` (default: `ffprobe`)
+  - Path to the `ffprobe` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+- `TONE_PATH` (default: `tone`)
+  - Path to the `tone` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+
+## Network
+
+- `HOST` (default: *empty*)
+  - The host Audiobookshelf binds to.
+    Most commonly, this will be `127.0.0.1` if you want the service to listen to localhost only,
+    or empty if you want to listen to all interfaces (both IPv4 and IPv6).
+- `PORT` (default: `3333`)
```

Comment:
The default for the debian package is `13378` and the default for docker is `80`. Currently the fallback to `3333` wouldn't be used unless you have that set in the rpm.

Reply:
You are right. In `index.js` the port is set to `80` by default.

On the Debian packages, only the configuration is set to `13378` by default though. I understand a default value to be the value used if no extra configuration is being provided. In that sense, I would have said it is `3333` there.

But since we don't have a common default, I think we should just remove the statement about the default here. That's just confusing prople who are seeing a different default on their particular installation.